docs/mkdocs.yml

```diff
@@ -90,11 +91,21 @@ nav:
       - Tinybird: https://www.tinybird.co/docs/forward/get-data-in/table-functions/iceberg
       - Trino: https://trino.io/docs/current/connector/iceberg.html
   - Catalogs:
-    - AWS Glue: aws/#glue-catalog
-    - AWS DynamoDB: aws/#dynamodb-catalog
-    - custom-catalog.md
+    - Apache Hadoop: https://iceberg.apache.org/javadoc/nightly/org/apache/iceberg/hadoop/HadoopCatalog.html
+    - Apache Hive: https://iceberg.apache.org/javadoc/nightly/org/apache/iceberg/hive/HiveCatalog.html
```

Comment:
You may link https://iceberg.apache.org/docs/nightly/hive/#global-hive-catalog

Reply:
Good idea, thanks. Changed 75d0c5193297e16839e340c4a38f0d713855a7b1